src/applications/mhv/secure-messaging/components/MessageDetailBlock.jsx

```diff
@@ -61,15 +61,18 @@ const MessageDetailBlock = props => {
     [location.pathname, dispatch],
   );
 
+  const categoryLabel = Categories[category];
+
   useEffect(
     () => {
       focusElement(document.querySelector('h1'));
+      document.title = `${categoryLabel}: ${subject} ${
+        PageTitles.PAGE_TITLE_TAG
+      }`;
```

Comment:
I am wondering if it would make sense to handle all titles for Thread Details view in `ThreadDetails.jsx` component, especially since you are already handling some scenarios there?